### DISCUSSION TITLE
Is the randomizer set up to always make each seed possible to beat, or will it scatter everything truly randomly

### DISCUSSION THREAD

**Discussion by TheCodeHermit** [phst-randomizer/ph-randomizer#467] [General]
Or at least what is planned, I understand that it may not be fully implemented. But is it planned to always be possible, or will that be a problem for users to deal with

**Comment by mike8699** [comment]
Similar to other zelda randomizer, it's set up to always be beatable. The implementation is very much a work-in-progress and certainly has some bugs in it, but at the moment we have the entire game mapped out using a custom graph-based DSL based on the logic for the minish cap randomizer (https://github.com/phst-randomizer/ph-randomizer/tree/main/ph_rando/shuffler/logic).
